app/helpers/billing_helper.rb

```diff
@@ -0,0 +1,17 @@
+module BillingHelper
+  private
+
+  def default_plan?(account)
+    installation_config = InstallationConfig.find_by(name: 'CHATWOOT_CLOUD_PLANS')
+    default_plan = installation_config.value.first
+    account.custom_attributes['plan_name'] == default_plan['name']
```

Comment:
```suggestion
    account.custom_attributes['plan_name'].nil? || account.custom_attributes['plan_name'] == default_plan['name']
```

We won't set up the stripe user until they go to the billing page.